ag_calibration.txt

```diff
@@ -0,0 +1,2 @@
+-1.0 -2.0 -2.0 
+-4.0 -5.0 -6.0 
```

Comment:
I'd prefer all calibration files be moved to a config file in `config` to avoid random .txt files floating around the repo